app/views/series/_form.html.erb

```diff
@@ -11,10 +11,12 @@
         </div>
     <% end %>
 
-    <div class="field">
-      Name<br/>
-      <%= @series.name %>
-    </div>
+    <% if request.path =~ /new/ %>
+      <div class="field">
+        <%= f.label :name, 'Name' %><br/>
+        <%= f.text_field :name, :size => 40 %>
```

Comment:
OK, ready for new review. As for collision with existing series, that was already in place (with unique index on name column)